Automattic/wp-calypso
# Add/jetpack scan component

#### Changes proposed in this Pull Request
This PR is more of an experiment gone sideways... 

It is an attempt to use the same entry point as calypso for Jetpack Cloud. 

This PR also adds a new `getApplication` selector which lets you know what kind of app you are currently running. 

The main reason for this is 
1. Increase the calypso jetpack cloud overlap.
2. It is super simple to add new calypso sections to the Jetpack Cloud or vice versa
3. We decrease the amount of code that gets loaded now on jetpack cloud. Since things are split up into sections.

#### Testing instructions
`npm start`
Load up calypso.localhost:3000 
notice if there are any regressions? 
Can you visit /scan or /backup sections? You should get a 404.

without restarting 
visit jetpack.cloud.localohost:3000/app
Notice that it loads as expected with the (jetpack logo) 
Notice that the new plugins section works as expected. 

Are you able to navigate to the rest of the sections as expected?
